arcjet-next/index.ts

```diff
@@ -124,6 +124,18 @@ export interface ArcjetNextRequest {
     | Partial<{ [key: string]: string }>;
 }
 
+interface ArcjetNextAppRequest extends ArcjetNextRequestCommon {
+  clone: () => Request;
+  body: ReadableStream<Uint8Array> | null;
+}
+
+interface ArcjetNextApiRequest extends ArcjetNextRequestCommon {
+  body: any;
+  clone?: never;
+}
```

Comment:
I don't like having these separate interfaces. Instead, I made the ArcjetNextRequest extremely generic and you have to figure out what is available.